txpool/txpool_test.go

```diff
@@ -69,11 +70,11 @@ func newTx(addr types.Address, nonce, slots uint64) *types.Transaction {
 }
 
 // returns a new txpool with default test config
-func newTestPool() (*TxPool, error) {
+func newTestPool(header *types.Header) (*TxPool, error) {
```

Comment:
Why this parameter? Not used anywhere, you pass "nil" for every func call...

Reply:
@Aleksao998 please address this comment

Reply:
@zivkovicmilos requested a change to make that we can adjust header when are creating the pool in our tests and not to have default values. I agree that this is not a pretty solution at all. But was the only one I have found. I will tag both of you in this discussion.

Reply:
What about adding a wrapper mock struct that embeds TxPool and has a method `SetHeader` or something like that?

Reply:
I am not really sure what do you think